docs/tutorials/getting_started.md

```diff
@@ -203,7 +203,7 @@ pcf/0*                       active    idle   10.1.10.146
 router/0*                    active    idle   10.1.10.145         
 self-signed-certificates/0*  active    idle   10.1.10.141         
 smf/0*                       active    idle   10.1.10.154         
-traefik/0*                   error     idle   10.1.10.160         hook failed: "ingress-relation-changed"
+traefik/0*                   error     idle   10.1.10.160         hook failed: "certificates-relation-changed"
```

Comment:
I think that the ingress relation is the one that fails because of the IP validation ?

Reply:
You are right. This is updated. We also need to fix it here: https://github.com/canonical/charmed-aether-sd-core/blob/main/docs/tutorials/getting_started.md
![image](https://github.com/user-attachments/assets/f766bb61-ee16-4447-96a6-19fd8fbce825)